testonly/mdm/mdmtest/main.go

```diff
@@ -53,6 +53,7 @@ var (
 	checkers        = flag.Int("checkers", 3, "Number of parallel checker goroutines to run")
 	emitInterval    = flag.Duration("emit_interval", 10*time.Second, "How often to output the summary info")
 	deadline        = flag.Duration("deadline", 60*time.Second, "Deadline for single add+get-proof operation")
+	minMergeDelay   = flag.Duration("min_merge_delay", 3*time.Second, "Minimum merge delay")
```

Comment:
Doc could be a bit more explanatory. Also is this really an MMD as it looks like a delay of some sort?

Reply:
Added some more text.   It's a *minimum* merge delay -- for (cloud)spanner storage layers, there's no point in checking for inclusion until after the read-staleness interval has passed.